gulpfile.js

```diff
@@ -42,6 +43,12 @@ gulp.task('fonts', function() {
     .pipe(gulp.dest('./dist/fonts'));
 });
 
+gulp.task('webpack-token-worker', function() {
+    return gulp.src('.src/js/workers/token-worker.js')
```

Comment:
I am getting Path doesn't exist, for some reason.

→ gulp webpack-token-worker
[19:22:30] Using gulpfile ~/tripleo_ui/gulpfile.js
[19:22:30] Starting 'webpack-token-worker'...

stream.js:94
      throw er; // Unhandled stream error in pipe.
            ^
Error: Path doesn't exist '/home/jtomasek/tripleo_ui/tripleo_ui_token_worker.js'
    at MemoryFileSystem.writeFileSync (/home/jtomasek/tripleo_ui/node_modules/gulp-webpack/node_modules/memory-fs/lib/MemoryFileSystem.js:175:10)
    at MemoryFileSystem.writeFile (/home/jtomasek/tripleo_ui/node_modules/gulp-webpack/node_modules/memory-fs/lib/MemoryFileSystem.js:220:8)
    at Tapable.writeOut (/home/jtomasek/tripleo_ui/node_modules/webpack/lib/Compiler.js:259:27)
    at Tapable.<anonymous> (/home/jtomasek/tripleo_ui/node_modules/webpack/lib/Compiler.js:245:20)
    at /home/jtomasek/tripleo_ui/node_modules/webpack/node_modules/async/lib/async.js:122:13
    at _each (/home/jtomasek/tripleo_ui/node_modules/webpack/node_modules/async/lib/async.js:46:13)
    at Object.async.each (/home/jtomasek/tripleo_ui/node_modules/webpack/node_modules/async/lib/async.js:121:9)
    at Tapable.emitFiles (/home/jtomasek/tripleo_ui/node_modules/webpack/lib/Compiler.js:234:20)
    at /home/jtomasek/tripleo_ui/node_modules/webpack/node_modules/mkdirp/index.js:48:26
    at Object.oncomplete (fs.js:108:15)